packages/manager/modules/iam/src/resolves/misc.resolve.js

```diff
@@ -79,6 +79,34 @@ statementResolve.resolves = [entityResolve];
 
 // ---------------------------------------------------------------------------------------------------- //
 
+/**
+ * Get the status of the advanced mode
+ * @returns {boolean}
+ */
+const advancedModeResolve = /* @ngInject */ ($q, PreferencesService) => {
+  return $q
+    .resolve(PreferencesService.getPreference(KEY_PREFERENCES.advancedMode))
```

Comment:
$q is not needed here
```suggestion
  return PreferencesService.getPreference(KEY_PREFERENCES.advancedMode)
```

Reply:
It's applied